packages/e2e-tests/specs/editor/plugins/cpt-locking.test.js

```diff
@@ -119,22 +119,22 @@ describe( 'cpt locking', () => {
 			);
 		} );
 
-		it( 'can use the global inserter in inner blocks', async () => {
+		it( 'should not allow blocks to be inserted in inner blocks', async () => {
 			await page.click( 'button[aria-label="Two columns; equal split"]' );
-			await page.click(
-				'.wp-block-column .block-editor-button-block-appender'
-			);
-			await page.type( '.block-editor-inserter__search input', 'image' );
-			await pressKeyTimes( 'Tab', 2 );
-			await page.keyboard.press( 'Enter' );
-			await page.click( '.edit-post-header-toolbar__inserter-toggle' );
-			await page.type(
-				'.block-editor-inserter__search input',
-				'gallery'
-			);
-			await pressKeyTimes( 'Tab', 2 );
-			await page.keyboard.press( 'Enter' );
-			expect( await page.$( '.wp-block-gallery' ) ).not.toBeNull();
+			expect(
+				await page.$(
+					'.wp-block-column .block-editor-button-block-appender'
+				)
+			).toBeNull();
+
+			expect(
```

Comment:
I have added a test to confirm that the global inserter is also disabled.

This code is the same as what we do in `shouldDisableTheInserter`, but for the sake of clarity, I have copied the code inside verbatim. Without the use of copy, I think it would be difficult to understand what you are trying to do as follows:

```javascript
it( 'should not allow blocks to be inserted in inner blocks', async () => {
	await page.click( 'button[aria-label="Two columns; equal split"]' );
	expect(
		await page.$(
			'.wp-block-column .block-editor-button-block-appender'
		)
	).toBeNull();

	// Hard to know if we are actually testing or trying to perform some action.
	shouldDisableTheInserter();
} );
```